src/main/resources/boggle_application.fxml

```diff
@@ -54,24 +53,32 @@
     </bottom>
 
     <fx:define>
-        <ArrayList fx:id="letterTiles">
-            <fx:reference source="tile1"/>
-            <fx:reference source="tile2"/>
-            <fx:reference source="tile3"/>
-            <fx:reference source="tile4"/>
-            <fx:reference source="tile5"/>
-            <fx:reference source="tile6"/>
-            <fx:reference source="tile7"/>
-            <fx:reference source="tile8"/>
-            <fx:reference source="tile9"/>
-            <fx:reference source="tile10"/>
-            <fx:reference source="tile11"/>
-            <fx:reference source="tile12"/>
-            <fx:reference source="tile13"/>
-            <fx:reference source="tile14"/>
-            <fx:reference source="tile15"/>
-            <fx:reference source="tile16"/>
-        </ArrayList>
+        <Array fx:id="letterSet">
```

Comment:
Are you sure you can declare a 2D array like this in the xml?

Reply:
No, and I don't see much documentation on it. I'll try to work on a different way around it.